notification-service-sdk/README.md

```diff
@@ -0,0 +1,144 @@
+# notification-service-sdk
+
+Parodos Notification Service API
+- API version: v1.0.0
+
+This is the API documentation for the Parodos Notification Service. It provides operations to send out and check notification. The endpoints are secured with oAuth2/OpenID and cannot be accessed without a valid token.
+
+
+*Automatically generated by the [OpenAPI Generator](https://openapi-generator.tech)*
+
+
+## Requirements
+
+Building the API client library requires:
+1. Java 1.7+
+2. Maven (3.8.3+)/Gradle (7.2+)
+
+## Installation
+
+To install the API client library to your local Maven repository, simply execute:
+
+```shell
+mvn clean install
+```
+
+To deploy it to a remote Maven repository instead, configure the settings of the repository and execute:
+
+```shell
+mvn clean deploy
+```
+
+Refer to the [OSSRH Guide](http://central.sonatype.org/pages/ossrh-guide.html) for more information.
+
+### Maven users
+
+Add this dependency to your project's POM:
+
+```xml
+<dependency>
+  <groupId>dev.parodos</groupId>
+  <artifactId>notification-service-sdk</artifactId>
+  <version>1.0.5-SNAPSHOT</version>
```

Comment:
Should we keep this version up to date as we release?

Reply:
This file is auto generated and the version is detrmined by the the version durring the auto generation (see here: https://github.com/parodos-dev/parodos/blob/cead9e12e393292e910b143982972ef6bcc57f4b/notification-service/pom.xml#L308)
Currently there is issue that it use the actual version and not the reference (see last commit description: https://github.com/parodos-dev/parodos/pull/156/commits/f40fcb3eca71b54b90efa70f4c08c5ca7f7862a5), I plan to investigate more what is the right way to use here, but prefer do do this in different PR, if its ok